src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC, useMemo, useState } from 'react';
+import './TodoList.scss';
+import { Todo, StatusFilter } from '../../types';
+import { filterTodos } from '../../utils';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[];
+  statusFilter: StatusFilter;
+}
+
+export const TodoList: FC<Props> = ({ todos, statusFilter }) => {
+  const [editingTodo] = useState<Todo | null>(null);
+  const [loadingTodo] = useState<Todo | null>(null);
```

Comment:
```suggestion
  const [loadingTodoIds] = useState<number[]>([]);
```
You can store only id, and in the next tasks you need to handle multiple loaders at the same time so you can store an array of ids and depending on it show loaders